packages/client/base/src/types/errors.ts

```diff
@@ -3,6 +3,7 @@ export const PaymentErrors = {
     PAYMENT_METHOD_INVALID: "payments:payment-method.invalid",
     EMAIL_INVALID: "payments:email.invalid",
     CLIENT_ID_INVALID: "payments:client-id.invalid",
+    MISSING_RECIPIENT: "payments:recipient.missing",
```

Comment:
should use required, not missing, to match headless

Reply:
+1